Iot/src/V1/Gapic/DeviceManagerGapicClient.php

```diff
@@ -1173,9 +1173,9 @@ public function listDeviceStates($name, array $optionalArgs = [])
      * ```
      * $deviceManagerClient = new DeviceManagerClient();
      * try {
-     *     $formattedResource = $deviceManagerClient->registryName('[PROJECT]', '[LOCATION]', '[REGISTRY]');
+     *     $resource = '';
```

Comment:
this seems like a documentation regression.

Reply:
This is likely related to the deprecation of the formatting methods, so should be expected.